src/tbb/misc_ex.cpp

```diff
@@ -257,7 +257,8 @@ int ProcessorGroupInfo::NumGroups = 1;
 int ProcessorGroupInfo::HoleIndex = 0;
 
 ProcessorGroupInfo theProcessorGroups[MaxProcessorGroups];
-
+int calculate_numa[MaxProcessorGroups];  //Array needed for FindProcessorGroupIndex
```

Comment:
Maybe add a better comment. Not that is just needed for that function but also what its values mean.